docs/reference/glossary.md

```diff
@@ -93,6 +93,27 @@ develop, ship, and run applications
 - The docker daemon process running on the host which manages images and containers
 
 
+## Docker for Mac
+
+[Docker for Mac](https://docs.docker.com/docker-for-mac/) is an easy-to-install,
+lightweight Docker development environment designed specifically for the Mac. A
+native Mac application, Docker for Mac uses the macOS Hypervisor framework,
+networking, and filesystem. It's the best solution if you want to build, debug,
+test, package, and ship Dockerized applications on a Mac. Docker for Mac
```

Comment:
@thaJeztah agreed re: s/on/from/ , I thought this when I was writing it. With regard to the expansion on "build, ship, and run", I am keying off of the web site content I'm working with @bettyjunod and @friism on.